src/main/resources/l10n/JabRef_zh.properties

```diff
@@ -38,19 +38,20 @@ Add_subgroup=添加子分组
 Add_to_group=添加到分组
 Added_group_"%0".=已添加分组_"%0"。
 Added_new=已添加
-Added_string=已添加字符串
+Added_string=已添加简写字串
 Additionally,_entries_whose_<b>%0</b>_field_does_not_contain_<b>%1</b>_can_be_assigned_manually_to_this_group_by_selecting_them_then_using_either_drag_and_drop_or_the_context_menu._This_process_adds_the_term_<b>%1</b>_to_each_entry's_<b>%0</b>_field._Entries_can_be_removed_manually_from_this_group_by_selecting_them_then_using_the_context_menu._This_process_removes_the_term_<b>%1</b>_from_each_entry's_<b>%0</b>_field.=此外，那些“<b>%0</b>”域里不包含“<b>%1</b>”的记录可以被手动添加到此分组（使用拖放或者右键菜单）——这个操作将会把词组“<b>%1</b>”添加到每条记录的“<b>%0</b>”域中。选中记录后用右键菜单可以将该记录从此分组中移除——这个操作将会把“<b>%1</b>”从被移除记录的“<b>%0</b>”域中去除。
 Advanced=高级
+Advanced_search_active.=高级搜索行为。
 Advanced_search_active.=
```

Comment:
Duplicates are no longer allowed. Please just add your translation in an existing line, do not create new key value pairs.